sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/sdk/controller/group/SDKThroughputGroupControllerBase.java

```diff
@@ -214,7 +214,7 @@ private String getResolvedPartitionKeyRangeId(RxDocumentServiceRequest request)
 
     private Mono<Boolean> shouldUpdateRequestController(RxDocumentServiceRequest request) {
         return this.partitionKeyRangeCache.tryGetRangeByPartitionKeyRangeId(
-                null, request.requestContext.resolvedCollectionRid, request.requestContext.resolvedPartitionKeyRange.getId(), null)
+                null, request.requestContext.resolvedCollectionRid, request.requestContext.resolvedPartitionKeyRange.getId(), null, new StringBuilder())
```

Comment:
Though its not relevant for this case, however, just wanted to note that this is different than what we have in PkRangesThroughputRequestController.java